cpu/stm32_common/periph/dac.c

```diff
@@ -27,6 +27,37 @@
 /* only compile this, if the CPU has a DAC */
 #ifdef DAC
 
+/* patch RCC clock bit in case there are 2 DAC's */
+#ifndef RCC_APB1ENR_DACEN
+#define RCC_APB1ENR_DACEN       RCC_APB1ENR_DAC1EN
+#endif
+
+/* patch the write data register macro based on 1 or 2 channel DAC's */
+#ifdef DAC_ONLY_1_CHANNEL
+
+#define DAC_WRITE_OUTPUT(line, value)       \
+    DAC->DHR12R1 = value;
+
+#else
+
+#define DAC_WRITE_OUTPUT(line, value)       \
+    if (dac_config[line].chan) {            \
+        DAC->DHR12R2 = value;               \
+    }                                       \
+    else {                                  \
+        DAC->DHR12R1 = value;               \
+    }
+
+#endif
+
+/**
+ * @brief   DAC line configuration data
+ */
+typedef struct {
+    gpio_t pin;             /**< pin connected to the line */
+    uint8_t chan;           /**< DAC device used for this line */
+} dac_conf_t;
```

Comment:
I guess this would make more sense to be put in the `stm32_common/include/periph_cpu_common.h`, as it is used no only in this file but in other headers (periph_conf.h files...).